net/socket/udp_socket_starboard.h

```diff
@@ -42,6 +42,8 @@
 
 namespace net {
 
+#if SB_API_VERSION <= 15
```

Comment:
Can be in a followup, but trunk only supports SB17 now so if a file is gated behind any API version < 17, we can delete it.